package.json

```diff
@@ -83,8 +83,7 @@
     "requirejs": "gulp requirejs",
     "generateDocumentation": "gulp generateDocumentation",
     "instrumentForCoverage": "gulp instrumentForCoverage",
-    "eslint": "gulp eslint",
-    "eslint-watch": "gulp eslint-watch",
+    "eslint": "eslint ./**/*.js ./**/*.html --cache",
```

Comment:
Like, the warning `File ignored because of a matching ignore pattern. Use "--no-ignore" to override` shouldn't fail travis